packages/greenbox-data/data/iotms.ts

```diff
@@ -1207,7 +1207,7 @@ export default [
     type: "campground",
   }, // A Guide to Burning Leaves
   {
-    id: 11363,
+    id: 11364,
```

Comment:
what kind of shoddy reviewer wouldn't notice this one SMH

Reply:
also correct 

https://kol.coldfront.net/thekolwiki/index.php/Items_by_number_(11300-11399)